css/cssom-view/support/scroll-behavior.js

```diff
@@ -85,3 +85,7 @@ function scrollWindow(scrollingWindow, scrollFunction, behavior, elementToReveal
   args.top = elementToRevealTop;
   scrollingWindow[scrollFunction](args);
 }
+
+function setScrollAttribute(scrollingElement, scrollAttribute, scrollValue) {
+  scrollingElement[scrollAttribute] = scrollValue;
```

Comment:
It seems this helper function is not really useful... Maybe it makes things more explicit but it also makes them more verbose. Anyway, I don't have strong opinion on this.

Reply:
Done